### PR TITLE
Flat file structure and turn off execute bits and introduce CI

### LINKED ISSUE #24: Add 3 basic zero-coverage tests to CI
1. syntax check
1. coding standard
1. static analysis

What do you think?

### PR DESCRIPTION
**Pull request by szepeviktor**
1. all source code went into `includes/`, see https://github.com/szepeviktor/disable-blog/tree/restructure
1. `chmod -x`
1. three basic tests in GitHub Actions including WPCS

@joshuadavidnelson Please see separate commits.

Fixes #24

### CODE REVIEW

**Comment by szepeviktor** [comment]
🎉 CI for PR-s :)

**Comment by joshuadavidnelson** [comment]
@szepeviktor I made an attempt to reconcile some other changes with your PR and looks like some are not clearing code standard (seems my VSCode default to spaces especially not happy). I couldn't get `phpstan` running locally so I'll have to circle back when I have more time later this week.

**Comment by szepeviktor** [comment]
I've found something for you https://stackoverflow.com/a/59353357

File: includes/class-disable-blog-admin.php [hunk 1]
```diff
@@ -328,7 +335,23 @@ public function remove_menu_pages() {
...
+     * Filter the body classes for admin screens to toggle on plugin specific styles.
+     *
+     * @param array $classes
+     * 
+     * @return array
+     */
+    function admin_body_class( $classes ) {
+
+        if( $this->has_front_page() )
+            $classes .= ' disabled-blog';
```
Comment (szepeviktor) [inline includes/class-disable-blog-admin.php]: Is it `$classes[] = ' disabled-blog';` ?

Comment (joshuadavidnelson) [reply includes/class-disable-blog-admin.php:350]: The [`admin_body_class`](https://developer.wordpress.org/reference/hooks/admin_body_class/) actually passes a string, whereas `post_class` and `body_class` pass an array. The `$classes .= ' disabled-blog';` is correct, but the doc block on that function was incorrectly calling it out as an array.

**Comment by szepeviktor** [comment]
VSCode must be able to understand https://editorconfig.org/
you can also commit the config file here

**Comment by szepeviktor** [comment]
@joshuadavidnelson What a big Step forward! 🏃‍♂️

**Pull request merged**